src/lib/challenge.ts

```diff
@@ -19,16 +19,20 @@
  - Name Surname <name.surname@gatesfoundation.com>
 
  - Abhimanyu Kapur <abhi.kapur09@gmail.com>
+ - Raman Mangla <ramanmangla@google.com>
 
  --------------
  ******/
 
 /* istanbul ignore file */
 
-// Flag to igore BDD testing, which will be dealt with in a later ticket
+/*
+ * Flag to ignore BDD testing, which will be dealt
+ * with in a later ticket
+ */
 
 import util from 'util'
-import crypto from 'crypto'
+import crypto, { KeyObject } from 'crypto'
```

Comment:
Hey why aren't you just using crypto.KeyObject to reference it? Instead of importing that separately?

Reply:
Yes, I could do that too.